assignments/assignment05.rb

```diff
@@ -6,10 +6,39 @@
 #  Problem 1 -  implement prop_reader, write MiniTest unit tests
 
 # expected results:
+
 class PropReader
-  prop_reader :flavor
+  attr :flavor
```

Comment:
if you want to test that `prop :flavor` works, why are you replacing this with `attr :flavor` ?